modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/http/ESHttpClient.java

```diff
@@ -106,9 +217,14 @@ public String getClusterName() {
 		String clusterName;
 		try {
 			JestResult result = client.execute(nodesinfo);
-			clusterName = result.getValue("cluster_name").toString();
+			if (result == null ) {
```

Comment:
@faxm0dem : but if result is null then calling `result.getValue()` is not the best idea :)